aea/cli/create.py

```diff
@@ -142,7 +108,7 @@ def create(click_context, agent_name, author, local):
             registry_path=os.path.join("..", DEFAULT_REGISTRY_PATH),
             description="",
         )
-        agent_config.default_connection = DEFAULT_CONNECTION
+        agent_config.default_connection = DEFAULT_CONNECTION  # type: ignore
```

Comment:
why is type ignore needed?

Reply:
It’s really weird. I din’t change anything in this line actually and even checked with debugger type of both expression and var and it’s correct. But static type check complains that it’s str, not PublicId.